specification/resources/resource-manager/Microsoft.Authorization/stable/2020-05-01/privateLinks.json

```
@@ -335,13 +384,67 @@
         },
         "x-ms-examples": {
           "Delete a Resource Management Private Link.": {
-            "$ref": "./examples/ListResourceManagementPrivateLink.json"
+            "$ref": "./examples/ListSubscriptionResourceManagementPrivateLink.json"
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Authorization/resourceManagementPrivateLinks": {
+      "get": {
+        "tags": [
+          "ResourceManagementPrivateLink"
+        ],
+        "operationId": "ResourceManagementPrivateLink_ListAtRGLevel",
```

Should this be `ListByResourceGroup`?
```suggestion
        "operationId": "ResourceManagementPrivateLink_ListByResourceGroup",
```